src/components/side_panel/chart/line_chart/line_chart_config_panel.xml

```diff
@@ -14,6 +14,18 @@
             Stacked linechart
           </label>
         </div>
+        <div class="o-checkbox">
+          <label>
+            <input
+              type="checkbox"
+              name="cumulated"
+              t-att-checked="props.definition.cumulated"
+              t-on-change="onUpdateCumulated"
+              class="align-middle"
+            />
+            Cumulated data
```

Comment:
```suggestion
            Cumulative data
```
I think that's more english. It's a stacked chart, but a cumulative chart, not cumulated chart.

I'd also rename all the occurences of `cumulated` by `cumulative` in the code/function/variables.

Reply:
Okay sure, updated requested changes.